extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java

```diff
@@ -43,6 +43,37 @@ public static FlywayDataSourceRuntimeConfig defaultConfig() {
     @ConfigItem
     public Optional<String> defaultSchema = Optional.empty();
 
+    /**
+     * The JDBC URL that Flyway will use to connect to the datasource.
+     * Falls back to the datasource URL if not specified.
+     */
+
+    @ConfigItem
+    public Optional<String> jdbcUrl = Optional.empty();
+
+    /**
+     * Sets the user that Flyway will use to connect to the datasource.
+     * Falls back to the datasource username if not specified.
+     */
+
+    @ConfigItem
+    public Optional<String> username = Optional.empty();
+
+    /**
+     * Sets the password that Flyway will use to connect to the datasource.
+     * Falls back to the datasource password if not specified.
+     */
+
+    @ConfigItem
+    public Optional<String> password = Optional.empty();
+
+    /**
+     * credentialsProviderName credentialsProviderName the name of the credentials provider.
+     */
+
+    @ConfigItem
+    public Optional<String> credentialsProviderName = Optional.empty();
+
```

Comment:
Unless you're implementing what @sberyozkin proposed in https://github.com/quarkusio/quarkus/pull/29756/files#r1043347239 I'd suggest to remove these lines